charter.html

```diff
@@ -178,24 +179,54 @@ <h3 id=chair-selection>Chair Selection</h3>
 <a href=https://www.w3.org/People/functions/w3m#dom>Dominique
 Hazaël-Massieux</a>.
 
-<h2 id=deliverables>Deliverables</h2>
+<h2 id=proposals>Proposals</h2>
```

Comment:
Is there a concrete difference between how the CG will interact with a Proposal vs a Work Item? It looks like a sufficiently complex proposal can get a repository before multiple engines have expressed interest, and the CG discusses them similarly.

Would it shorten or simplify the charter to say that everything's a Proposal, and the chairs will keep a list of proposals for which 2 engines have expressed interest?

Reply:
The major difference seems to be around the allocation of [potentially scarce] Editors, which are appointed by the chairs (do these editors have to be members of the CG?) to advance the work item. The Editors then takeover much of the responsibility of "directing the work" until they judge its ready to advance beyond the CG. Did I get that right?

What does this mean for proposals that have a highly-engaged and willing "owner"/"creator"? If the chairs do not want to see the proposal advance (or the proposal doesn't meet the implementer criteria), I'm not sure they could do much to stop a pro-active proposal owner (not an officially minted editor) from carrying on. It seems the purpose of promotion to Work Item then is to lend a helping hand to good proposals with less-active owners and get them moving even faster in the process. Might be worth calling out the rationale a bit more.

Reply:
Hmm. The intent is that the output of the proposal phase is an explainer, and the output of the work item phase is a revised explainer and a draft spec. This is spelled out in the proposals repo https://github.com/privacycg/proposals but not in the charter. I'll try to figure out how to incorporate that here. Thanks!

Reply:
I'm going to raise this as a separate issue.